python/torch_mlir_e2e_test/test_suite/__init__.py

```diff
@@ -10,6 +10,7 @@
     "QuantizedMLP_basic",
 }
 
+
```

Comment:
Just out of curiosity, are these changes from applying an auto-formatting tool?

Reply:
Yes, yapf.

Reply:
I see, thanks much!  It looks like this was long overdue.

Minor comment: in the future, let's keep the formatting changes in a separate commit (in the same PR) to keep the diffs that need to be reviewed short.

Reply:
The same is followed in this PR. I have added the formatting changes in a separate commit.

Reply:
Oh shoot, I am so sorry!  My apologies, I should have clicked the commit list instead of my looking at the changed file.  Sorry, that was completely my fault.